code-snippet-plugin/src/main/java/com/okta/maven/snippet/CodeSnippetMojo.java

```diff
@@ -115,15 +118,30 @@ public void execute() throws MojoExecutionException {
 
         final Map<String, String> methodBodyMap = new HashMap<>();
 
-
         new VoidVisitorAdapter<Object>() {
             @Override
             public void visit(final MethodDeclaration method, final Object arg) {
                 super.visit(method, arg);
 
                 String body = method.getBody().get().getStatements().stream()
-                        .map(Node::toString)
+                        .map(statement -> {
+                            TokenRange tokenRange = statement.getTokenRange().get();
```

Comment:
Just out of curiosity, why does this method require an extra `get`?

Reply:
It is a Java Optional. Something like:
```java
Optional<String> myValue = ...
String actualValue = myValue.get(); // gets the value or throws an exception
```
It is a functional technique to not null check all over the place. In my case I'm not handling the error because i want the build to fail if there isn't anything to `get`.